test/credentials_test.dart

```diff
@@ -104,8 +104,6 @@ Future<void> main([List<String>? args]) async {
   baasTest('Email/Password - confirm user token expired', (configuration) async {
     final app = App(configuration);
     final authProvider = EmailPasswordAuthProvider(app);
-    String username = generateRandomEmail();
```

Comment:
Isn't this already deleted in main?

Reply:
if it is it won't matter when it is merged

Reply:
Yes, I'll rebase